pkg/lib/aws/ecr.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"encoding/base64"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/service/ecr"
+	"github.com/cortexlabs/cortex/pkg/lib/errors"
+	"github.com/cortexlabs/cortex/pkg/lib/regex"
+)
+
+type ECRAuthConfig struct {
+	Username      string
+	AccessToken   string
+	ProxyEndpoint string
+}
+
+func (c *Client) GetECRAuthToken() (*ecr.GetAuthorizationTokenOutput, error) {
+	svc := c.ECR()
```

Comment:
nit: you can remove this line and use `c.ECR()` below

Reply:
Done!